server/auth/types/basic/routes.ts

```diff
@@ -203,14 +209,19 @@ export class BasicAuthRoutes {
             expiryTime: Date.now() + this.config.session.ttl,
           };
 
-          if (this.config.multitenancy?.enabled) {
+          if (user.multitenancy_enabled) {
```

Comment:
Below you have `multitenancy_enabled?` but here it is not optional. Is this intentional? Just want to make sure you meant them to be different.

Reply:
Yes